service/policy/db/queries/obligations.sql

```diff
@@ -51,6 +51,29 @@ LEFT JOIN inserted_values iv ON iv.obligation_definition_id = io.id
 GROUP BY io.id, io.name, io.metadata, n.id, fqns.fqn;
 
 -- name: getObligation :one
+WITH obligation_triggers_agg AS (
+    SELECT
+        ot.obligation_value_id,
+        JSON_AGG(
+            JSON_BUILD_OBJECT(
+                'id', ot.id,
+                'action', JSON_BUILD_OBJECT(
+                    'id', a.id,
+                    'name', a.name
+                ),
+                'attribute_value', JSON_BUILD_OBJECT(
```

Comment:
We should add a follow-up to do a migration to index all of these relations for faster reads.